ucte-network/src/test/java/eu/itesla_project/ucte/network/UcteFileReadWriteTest.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.ucte.network;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import eu.itesla_project.ucte.network.io.UcteReader;
+import eu.itesla_project.ucte.network.io.UcteWriter;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Iterator;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Christian Biasuzzi <christian.biasuzzi@techrain.it>
+ */
+public class UcteFileReadWriteTest {
```

Comment:
Is it possible to extend ConverterBaseTest (IPST-core commons module) and call roundTripTest method?